src/lib/i18n.ts

```diff
@@ -32,6 +34,7 @@ i18next.init({
 					content: 'Content',
 					prompt: 'Prompt',
 					run: 'Run',
+					saveAndRun: 'Save & Run',
```

Comment:
- Should move the `saveAndRun` key to the root of the tree. Moving forward we want to avoid grouping keys.
- Should be capitalized: `Save & run`